scripts/service/timeless_pg.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=PostgreSQL DB Timeless
+Documentation=man:postgres(1)
+
+[Service]
+Type=forking
+User=postgres
+#pg_conftool show -s data_directory;
+#Environment=PGDATA=/var/lib/postgresql/10/main
+ExecStart=/usr/lib/postgresql/10/bin/postgres -D /etc/postgresql/10/main
```

Comment:
@marceloamadeu How did you check that `/etc/postgresql/10/main` is PG_DATA directory?

Reply:
@vladarefiev These are two comments that I used in development. Removed now.